feeder/util/mqtt/client.py

```diff
@@ -155,7 +155,7 @@ async def send_cmd_schedule(
         )
 
     async def start(self):
-        await self.connect("mqtt://localhost:1883/")
+        await self.connect("mqtt://%s:%s@localhost:1883/" % (local_username, local_password))
```

Comment:
You can just use a format string here: `f"mqtt://{local_username}:{local_password}@localhost:1883/`

I’ve always found them about 100x easier to grok when trying to understand code.